modules/openapi-generator/src/main/resources/dart2/api_client.mustache

```diff
@@ -117,24 +118,24 @@ class ApiClient {
 
     try {
       switch(method) {
-        case 'POST': return await _client.post(url, headers: nullableHeaderParams, body: msgBody);
-        case 'PUT': return await _client.put(url, headers: nullableHeaderParams, body: msgBody);
-        case 'DELETE': return await _client.delete(url, headers: nullableHeaderParams);
-        case 'PATCH': return await _client.patch(url, headers: nullableHeaderParams, body: msgBody);
-        case 'HEAD': return await _client.head(url, headers: nullableHeaderParams);
-        case 'GET': return await _client.get(url, headers: nullableHeaderParams);
+        case 'POST': return await _client.post(url, headers: nullableHeaderParams, body: msgBody,);
+        case 'PUT': return await _client.put(url, headers: nullableHeaderParams, body: msgBody,);
+        case 'DELETE': return await _client.delete(url, headers: nullableHeaderParams,);
+        case 'PATCH': return await _client.patch(url, headers: nullableHeaderParams, body: msgBody,);
+        case 'HEAD': return await _client.head(url, headers: nullableHeaderParams,);
+        case 'GET': return await _client.get(url, headers: nullableHeaderParams,);
```

Comment:
Some use dartfmt to reformat the generated code, this would make it look nicer for them. Same below.